src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,16 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger();
+
     @Override
     public Order completeOrder(Long userId) {
         // TODO: add log message about method completeOrder was called
```

Comment:
The TODO comment should be removed before submitting the solution, as per the checklist requirement: 'Let's remove all TODO comments before submitting solution.'